src/pages/songayoung/Login/AyLogin.js

```diff
@@ -1,5 +1,85 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
 
 export default function AyLogin() {
-  return <h1>HappyLogin</h1>;
+  const check = () => {
+    fetch('http://10.58.52.237:3000/auth/signin', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json;charset=utf-8' },
+      body: JSON.stringify({
+        email: email,
+        password: password,
+      }),
+    })
+      .then(response => response.json())
+      .then(data => console.log(data));
+  };
+
+  const [email, setId] = useState('');
+  const [password, setPw] = useState('');
```

Comment:
```suggestion
  const [email, setEmail] = useState('');
  const [password, setPassword] = useState('');
```

setState의 이름은 해당 state를 변경하는 함수라는 의미가 담기도록 위와 같이 작성해 주어야 합니다 (컨벤션)